include/RAJA/util/View.hpp

```diff
@@ -120,7 +120,12 @@ struct View {
 // returns linear index of layout(ar...)
 template <typename Lay, typename Tup, camp::idx_t... Idxs>
 RAJA_HOST_DEVICE RAJA_INLINE 
-RAJA::Index_type selecttuple( Lay lyout, Tup&& tup, camp::idx_seq<Idxs...> )
+auto selecttuple( Lay lyout, Tup&& tup, camp::idx_seq<Idxs...> ) ->
```

Comment:
Wow. We're shortening LAYOUT/layout to Lay/lyout and Tuple/tuple to Tup/tup.
Are folks getting tire of typing 😃

Reply:
😃 I originally had these helper functions in the MultiView struct itself, and wanted to distinguish between the various Layouts and Tuples already in the struct. Would you prefer changing them back to their proper spellings?

Reply:
Nah. Brevity is the spice of life.